File: Python Word examples/tables/add_table_cell.py
import spirecloudword
from spirecloudword.api.tables_api import TablesApi
from spirecloudword.configuration import Configuration as WordConfiguration

appId = "your id"
appKey = "your key"

configuration = WordConfiguration(appId, appKey)
api = spirecloudword.api.tables_api.TablesApi(configuration)

filename = "Template.docx"
table_row_path = "sections/0/tables/0/rows/5"
index_of_table_cell = "1"
inputFolder = "input"
outputFilePath = "output/Tables/add_table_cell.docx"
result = api.add_table_cell(filename, table_row_path, index_of_table_cell=index_of_table_cell, folder=inputFolder, dest_file_path=outputFilePath)

File: JavaScript Word Examples/TablesApiDemo/Cells/addTableCell.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.TablesApi(apiClient); 

    var filename = "AddTable.docx";
    var tableRowPath = "Section/0/Body/0/Table/0/TableRow/0";
    var opts = {
     folder: 'Tables'
    };
    var desFilePath = "output/Tables/AddTableCell_1.docx";
    instance.addTableCell(filename, tableRowPath, desFilePath, opts,
    function (error, data, response) {
         if (error) throw error;
    });
})();